src/track/cue-point.js

```diff
@@ -27,6 +27,7 @@ class CuePoint {
 CuePoint.TYPE = {
   ID3: 'id3',
   EMSG: 'emsg',
+  CUE_POINT: 'cuepoint',
```

Comment:
CuePount.Type.CUE_POINT?

Reply:
agree it's wired but this is how cuepoint manager works. it generates a cue point of type 'cuepoint'.